BinaryTree.java

```diff
@@ -0,0 +1,165 @@
+package com.katermar;

```

Comment:
You might consider creating subpackages for each assignment to clearly separate code for each of them and to avoid naming conflicts (BinaryTree.TreeNode vs TreeNode from previous assignment).